src/sentry/web/frontend/setup_wizard.py

```diff
@@ -145,6 +145,14 @@ def post(self, request: HttpRequest, wizard_hash=None) -> HttpResponse:
         default_cache.set(key, cache_data, SETUP_WIZARD_CACHE_TIMEOUT)
         return HttpResponse(status=200)
 
+    @allow_cors_options
+    def options(self, request, *args, **kwargs):
+        """Handle responding to requests for the OPTIONS HTTP verb."""
+        response = HttpResponse()
+        response.headers["Allow"] = ", ".join(self._allowed_methods())
+        response.headers["Content-Length"] = "0"
+        return response
```

Comment:
Would it make sense to call `super().options()` instead of copying parts of the logic?

Reply:
yup